src/app/components/entrance-transition/index.js

```diff
@@ -1,12 +1,13 @@
 'use strict';
 
 import React from 'react';
+import ReactDOM from 'react-dom';
 import classnames from 'classnames';
 
 const EntranceTransition = React.createClass({
   componentDidMount() {
     this.timeout = setTimeout(() => {
-      React.findDOMNode(this).classList.add('show');
+      ReactDOM.findDOMNode(this).classList.add('show');
```

Comment:
Facebook is recommending against using `findDOMNode` (see [here](https://facebook.github.io/react/docs/react-dom.html#finddomnode)), you might want to consider migrating this pattern in the codebase to refs instead.